projects/components/src/filter-button/filter-button.service.ts

```diff
@@ -19,10 +19,18 @@ export class FilterButtonService {
     return this.filterBuilderService.lookup(attribute).buildFiltersForAvailableOperators(attribute, value);
   }
 
+  public isSupportedOperator(attribute: FilterAttribute, operator: UserFilterOperator): boolean {
+    return this.filterBuilderService.lookup(attribute).supportedOperators().includes(operator);
+  }
+
   /*
    * TODO: Combine service with filter-bar.service. A few duplicated methods here.
```

Comment:
PR to refactor file structure and combine service methods (comment on line 27) to follow but wanted to keep this PR scope down.